README.md

```diff
@@ -20,7 +20,9 @@ Those are common with the GTK 3 and GStreamer bindings and are part of the [gtk-
 
 For more information about each crate, please refer to their `README.md` file in their directory.
 
-__Required Rust version__: 1.48+
+## Minimum supported Rust version
+
+Currently, the minimum supported Rust version is `1.48.0`.
```

Comment:
We also have that in other sub-readmes so that the crates.io pages have that information, do you mind updating those as well to keep things consistent?

Reply:
Sure!